Src/PerformanceCollector/Shared/Implementation/WebAppPerformanceCollector/SdkVersionAzureWebApp.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.PerfCounterCollector.Implementation
+{
+    using Web.Implementation;
+
+    internal class SdkVersionAzureWebApp
+    {
+        internal static string sdkVersionAzureWebApp = VersionPrefix + GetAssemblyVersion();
+
+        internal const string VersionPrefix = "azwapc: ";
+
+        internal static string GetAssemblyVersion()
+        {
+            return null;
+            //return SdkVersionUtils.GetAssemblyVersion();
```

Comment:
This is the part of the logic that goes to PerfCollectorModule. We will control the assembly version based on the platform there, and assign it to the client context.